src/main/studio/utils/WindowsAppUserMode.java

```diff
@@ -5,16 +5,16 @@
 import com.sun.jna.WString;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
-import studio.kdb.Config;
 import studio.kdb.Lm;
 import studio.ui.Util;
+import studio.utils.log4j.EnvConfig;
 
 public class WindowsAppUserMode {
     private static final Logger log = LogManager.getLogger();
 
     private final static boolean initialized = init();
 
-    private final static String mainID = "kdbStudioAppID" + Config.getEnvironment() + Lm.version;
+    private final static String mainID = "kdbStudioAppID" + EnvConfig.getEnvironment() + Lm.version;
```

Comment:
not sure what this class is used for - but is it ok if environment is `null` ?

Reply:
Basically this PR doesn't change the behaviour of the class. Before the change, it also can get Config.getEnvironment() = null in which case the code doesn't throw an Exception but use "null" as a String representation.

The purposes of the code is to use SetCurrentProcessExplicitAppUserModelID function from Win32 API (it works only in Windows). Windows groups icons of similar windows in the task bar. By default the grouping is based on the exe file initiated the window. So in case of Java, all Java frames initiated by the specific Java installation are grouped into a single icon in the Windows task bar.

The class sets the grouping logic for Windows OS. Namely, it separates StudioPanel from Chart icons. And also it separates frames which were executed with different env and/or different kdbStudio version. 

Technically this code doesn't differentiate absence of env (the default mode) with env=null.

Reply:
Thanks for the explanation